docs/static/releasenotes.asciidoc

```diff
@@ -3,81 +3,46 @@
-* Fixed  https://github.com/elastic/logstash/pull/8226[password support] in modules
-* A variety of small consistency / testing fixes for the Windows platform
-* Misc stability fixes
-
-[[logstash-6-0-0-beta2]]
-=== Logstash 6.0.0-beta2 Release Notes
-
+* Dropped support of the LS_JVM_OPTS env var
+* Dropped support of the USE_RUBY and USE_DRIP environment variables
 * Added `cloud.id` and `cloud.auth` support for modules to provide an easy way to integrate with Elastic Cloud service.
-* Added an explicit `--setup` phase for modules which is used to install dashboards and Elasticsearch template.
 * Enabled DLQ support for multipipelines
-
-[[logstash-6-0-0-beta1]]
-=== Logstash 6.0.0-beta1 Release Notes
-
-* Added new `logstash.yml` setting: `config.support_escapes`. When enabled, Logstash will interpret escape sequences in
-  strings in the pipeline configuration.
-* Breaking: The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s`, etc for
-  convenience. Previously, users had to convert this to a second time value themselves.
-* Breaking: The list of plugins bundled with Logstash 6.0.0-beta1 release has changed. Please consult the breaking changes document
-  for a complete list.
 * Added an {xpack} feature to manage Logstash configurations centrally in Elasticsearch. We've also added a UI to manage
   configurations directly in Kibana without having to restart Logstash.
 * Users can now visualize the Logstash configuration pipeline as part of the {xpack} monitoring feature.
-* We now report an error if config file referenced in the `pipelines.yml` is missing.
-
-[[logstash-6-0-0-alpha2]]
-=== Logstash 6.0.0-alpha2 Release Notes
-
-* Breaking: Currently, when Logstash is installed and set up via package managers, it loads all files found
-  in `/etc/logstash/conf.d` as configuration. This can be problematic if there are non-configuration files in this
-  folder. Starting from alpha2, Logstash will only glob files ending with `.conf` extension in `/etc/logstash/conf.d` ({lsissue}6979)[Issue 6979)]).
-* Added support for running multiple pipelines in the same Logstash instance. Running multiple pipelines
-  allow users to isolate data flow, provide separate runtime pipeline parameters and helps simplify complex
-  configurations.
-* A new `pipelines.yml` configuration file has been added to define pipelines.
-
-[[logstash-6-0-0-alpha1]]
-=== Logstash 6.0.0-alpha1 Release Notes
-
-* Introducing a new internal representation for the existing Logstash configuration that forms the
-  foundation of many new features. This is not a breaking change, and existing configs will work as is.
-
-[float]
 
-==== Filter Plugins
+==== Plugins
 
-*`GeoIP`*: You can now use MaxMind's commercial database to get enriched Geo information. ASN data can be
+*`GeoIP Filter`: You can now use MaxMind's commercial database to get enriched Geo information. ASN data can be
```

Comment:
If you want this to render as a bulleted list,  you need a space after the asterisk.

Reply:
GLOBAL COMMENT: After reading through the full list, I see that sometimes we use periods at the end of a bulleted item, and sometimes we don't. I don't want to go back and edit all my comments, but we should probably be consistent and just use a period everywhere.